### DISCUSSION TITLE
why my async combobox keeps loading the page until the end of the page?

### DISCUSSION THREAD

**Discussion by laptopmutia** [josefarias/hotwire_combobox#201] [Q&A]
I'm using pagy btw, it seems async combobox could override my pagination

lets say if my pagination have 1 object per page, async combobox keeps loading it until it have full length for the selections height, how to change this?

**Comment by josefarias** [comment]
Hey @laptopmutia could you share your code please?

Is the combobox loading the same page over and over, or is it requesting subsequent pages until it fills up the listbox height?

The latter would be expected. The former would suggest you might not be providing the `next_page` param as described in the docs.

Or it could be something else! I'd need more information to tell.

**Reply by laptopmutia** [reply]
I think its the latter, and I think I already fine now
